celery/worker/consumer/consumer.py

```diff
@@ -622,9 +622,9 @@ def on_task_received(message):
                 try:
                     ack_log_error_promise = promise(call_soon, (message.ack_log_error,),
                                                     on_error=self._restore_prefetch_count_after_connection_restart)
-                    reject_log_error_promise = promise(call_soon,
-                                                       (message.reject_log_error,),
-                                                       on_error=self._restore_prefetch_count_after_connection_restart)
+                    reject_log_error_promise = \
```

Comment:
Is there something we can do with this code formatting? It's not readable.

Reply:
agree, I will improve that
whether there are plans to use something like black?

Reply:
improved formating